rmgpy/reaction.py

```diff
@@ -1437,6 +1437,35 @@ def generate_high_p_limit_kinetics(self):
         """
         raise NotImplementedError("generate_high_p_limit_kinetics is not implemented for all Reaction subclasses.")
 
+    def reaction_to_html(self, surface_site_density=None):   
```

Comment:
> Thanks for fixing this Chris! Looks good to me. I just had one nit-picky comment about calling the function `kinetics_to_html` instead of `reaction_to_html`. Also, I think we could try to cythonize this function by adding it to `reaction.pxd` and declaring these in the function
> 
> ```
>         cdef double T
>         cdef str string
>         cdef list Tdata
> ```

thanks for looking at it. I don't think I can add the ```cdef double T``` etc. directly in the function in the .py file because it is not a  .pyx file (I get a compile error saying as much).
<img width="541" alt="image" src="https://user-images.githubusercontent.com/56306881/119699552-49200c00-be20-11eb-8eba-87180ca0cc0a.png">

 Is there another way to do this? Can variables defined in the .pxd be local to a specific function?

Reply:
🤔 maybe:
```
        cython.declare(T=cython.double, sting=str, Tdata=cython.list)
```

Reply:
that works, after timing it it seems like that is slightly speeding it up so I am assuming it is working as intended.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/56306881/119703130-38719500-be24-11eb-8719-2dd95214fdf5.png">

Reply:
lol, well faster is better than slower I guess